applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -62,14 +59,17 @@ struct subscriber {
 	const void *id;
 	uint32_t enabled_reports_bm;
 	struct enqueued_reports enqueued_reports;
+	uint8_t out_reports[ARRAY_SIZE(output_reports)][OUTPUT_REPORT_MAX_LEN];
```

Comment:
please hide it behind some type for simplification:
btw - if we use packed we can enable explicit access to report_id

```
#define OUTPUT_REPORT_DATA_MAX_LEN (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT)?(REPORT_SIZE_KEYBOARD_LEDS):(0))
...
struct report_data {
   uint8_t report_id;
   uint8_t data[OUTPUT_REPORT_DATA_MAX_LEN];
} __packed;

struct subscriber {
   ...
   struct report_data[ARRAY_SIZE(output_reports)];
   ...
};
```